flexget/components/thetvdb/api_tvdb.py

```diff
@@ -257,10 +257,12 @@ def get_posters(self):
         if not self._posters:
             log.debug('Getting top 5 posters for series %s', self.name)
             try:
+                poster_main = TVDBRequest().get('series/%s' % self.id).get('poster')
```

Comment:
please use fstring

Reply:
Sure thing. Only this new line or should I go ahead and convert all other `%s` replacements on the file for f-strings too?

Reply:
All please 😁

Reply:
On it 👍 .
Just one more question (I hope): Leave log entries alone to let it handle the parameters internally or convert them to f-strings too?